src/WebJobs.Script/Utility.cs

```diff
@@ -692,6 +694,19 @@ public static void ExecuteAfterDelay(Action targetAction, TimeSpan delay, Cancel
             }, TaskContinuationOptions.OnlyOnRanToCompletion);
         }
 
+        public static void ExecuteWithColdstartDelay(IEnvironment environment, Action targetAction, CancellationToken cancellationToken = default)
```

Comment:
Creating a single explicitly named helper that we can use for all these coldstart delays, so we're handling it consistently everywhere.

Reply:
Great! Shouldn't it be ExecuteWithColdStartDelay? :-)

---
In reply to: [384734964](https://github.com/Azure/azure-functions-host/pull/5697#discussion_r384734964) [](ancestors = 384734964)

Reply:
Renamed